app/code/community/Bolt/Boltpay/etc/system.xml

```diff
@@ -355,11 +355,39 @@ This function is called when the Bolt checkout modal is closed.
               <show_in_store>0</show_in_store>
               <sort_order>106</sort_order>
             </close>
+            <add_button_to_minicart translate="label">
+              <label>Add Bolt Button to the MiniCart Popup</label>
+              <frontend_type>select</frontend_type>
+              <source_model>adminhtml/system_config_source_yesno</source_model>
+              <sort_order>120</sort_order>
+              <show_in_default>1</show_in_default>
+              <show_in_website>1</show_in_website>
+              <show_in_store>0</show_in_store>
+            </add_button_to_minicart>
+            <replace_minicart_button_selectors>
```

Comment:
Do we need to distinguish `replace_minicart_button_selectors` and `selectors` at line 195? Maybe it's simpler to always use `selectors`?